app/html/newtab.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>New Tab</title>
+
+        <!-- Add my css file-->
+        <link rel="stylesheet" href="../styles/styles.css">
+    </head>
+
+    <body id="body">
+
+        <!-- Create Time / Date "widget"-->
+        <div id="datetime"></div>
+
+        <!-- Create Notepad-->
+            <textarea id="notepad" contenteditable="true" rows="6"></textarea> <br />
+            <button id="clearbtn" type="submit">Clear Note</button>
+
+       <!-- Helpful Links-->
+       <div class="links">
+           <ul>
+                <h2>Productivity Links</h2>
+                <div id="productivity">
+                    <!--Link #1 -->
+                    <a href="https://www.habitica.com">
+                        <li> <img class="icon" id="habitica" src="../images/habitica.png"></li>
+                    </a>
+
+                    <!--Link #2 -->
+                    <a href="https://www.todoist.com">
+                        <li><img class="icon" src="../images/todoist.jfif"></li>
+                    </a>
+
+                    <!--Link #3-->
+                    <a href="https://www.habitify.com"> 
+                        <li><img class="icon" src="../images/habitify.png"></li>
+                    </a>
+                </div>
+
+                <h2> Web Development Links</h2>
+                <div id="dev">
+                    <!--Link #4 -->
+                    <a href="https://www.w3schools.com"> 
+                        <li><img class="icon" src="../images/w3schools.jfif"></li>
+                    </a>
+
+                    <!--Link #5-->
+                    <a href="https://www.stackoverflow.com">
+                        <li><img class="icon" id="stack" src="../images/stackoverflow.png"></li> 
+                    </a>
+                </div>
+                
+           </ul>
+       </div>
+        <script src="../scripts/script.js"></script>
+    </body>
+</html>
```

Comment:
You are overcommenting a little bit. You don't need to add comments for every element, usually you can get away with letting the code speak for itself. Comments should only be used to provide extra context for code and, in the (hopefully rare) case where the code is messy, to explain what the code is doing.